Mobile Buy SDK/BuyTV/Info.plist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this is identical as well.  You can specify the existing plist from Build Settings->Info Plist File